src/automerge-repo-undo-redo.ts

```diff
@@ -216,6 +216,10 @@ export class AutomergeRepoUndoRedo<T> {
         });
 
         if (heads) {
+          if (stack.redos.length > 0) {
```

Comment:
this fixes the `can apply consecutive redos on untracked text` test and it does makes sense in my head I guess